omou-frontend/src/components/FeatureViews/Scheduler/SessionView.js

```diff
@@ -64,72 +65,160 @@ class SessionView extends Component {
 
 
     componentWillMount() {
-        this.setState(() => {
 
+        this.setState(() => {
+            const course = this.props.courses[this.props.computedMatch.params.course_id];
             let sessionData = this.props.courseSessions[this.props.computedMatch.params.course_id][this.props.computedMatch.params.session_id]
             let courseData = this.props.courses[this.props.computedMatch.params.course_id]
-
-            let instructorData = this.props.instructors
-
-
-
+            const instructor = this.props.instructors[course.instructor_id];
+            console.log(instructor)
             return {
                 sessionData: sessionData,
-                courseData: courseData
+                courseData: courseData,
+                instructor: instructor
             }
         })
+    }
 
 
+
+    // Formatting the date in the view
+    formatDate = (start, end) => {
+        let DayConverter = {
+            1: "Monday",
+            2: "Tuesday",
+            3: "Wednesday",
+            4: "Thursday",
+            5: "Friday",
+            6: "Saturday",
+        };
+        let MonthConverter = {
+            0: "January",
+            1: "February",
+            2: "March",
+            3: "April",
+            4: "May",
+            5: "June",
+            6: "July",
+            7: "August",
+            8: "September",
+            9: "October",
+            10: "November",
+            11: "December"
+        }
+
+        const date = new Date(start)
+        const dateNumber = date.getDate()
+        const dayOfWeek = date.getDay()
+        const startMonth = date.getMonth()
+        // Gets days
+        let Days = DayConverter[dayOfWeek]
+
+        //Gets months
+        let Month = MonthConverter[startMonth]
+
+        //Start times and end times variable 
+        let startTime = start.slice(11)
+        let endTime = end.slice(11)
+
+        // Converts 24hr to 12 hr time 
+        this.timeConverter = (time) => {
+            let timeString = time;
```

Comment:
Useless assignment, you can just declare this on line 130 (you assign it a value without ever using the initialized value here).